hardware_integration/include/isobus/hardware_integration/can_hardware_interface.hpp

```diff
@@ -145,7 +145,7 @@ class CANHardwareInterface
 	};
 
 	/// @brief A hard-coded update interval for the CAN stack. Mostly arbitrary
-	static const std::uint32_t CANLIB_UPDATE_RATE = 4;
+	static const std::uint32_t CANLIB_UPDATE_RATE = 10;
```

Comment:
Added `set_can_driver_update_period(std::uint32_t value)` function to `CANHardwareInterface` since building with PlatformIO kind of skips the stack's CMake files. So I think a setter function is preferred aswell.